test/test_transformers.py

```diff
@@ -1452,7 +1452,49 @@ def test_mem_efficient_fail_sm90(self, device):
 
 
 class TestSDPA(NNTestCase):
-    """ Used to test the functionality of scaled_dot_product_attention
+    """ Used to test generic functionality of scaled_dot_product_attention
+    Summary:
+        If you are adding a new test to this class, make sure that it runs
+        for both cpu and cuda. If you're test is only applicable to cuda,
+        add it to TestSDPACudaOnly.
+    """
+    @parametrize("contiguous_inputs", [True, False])
+    def test_sdp_math_gradcheck(self, device, contiguous_inputs: bool):
```

Comment:
I think this function wasn't deleted below, it's [here](https://github.com/pytorch/pytorch/blob/0de36bc9b05e1b6c869de39b59b1d14189a69a90/test/test_transformers.py#L1735)

Reply:
ahh thank you!